modules/web/src/app/settings/admin/dynamic-datacenters/datacenter-data-dialog/template.html

```diff
@@ -187,8 +187,12 @@
     <km-machine-flavor-filter class="km-machine-flavor-filter"
                               [formControl]="form.get(controls.MachineFlavorFilter)"
                               [machineFlavorFilter]="machineFlavorFilter"></km-machine-flavor-filter>
+    <ng-container *ngIf="form.get(controls.Provider).value === Provider.OpenStack">
+      <mat-checkbox [formControlName]="controls.EnableConfigDrive">
+        Enable Config Drive
+      </mat-checkbox>
+    </ng-container>
```

Comment:
I think we should add this setting under `Provider Configuration` heading.

Reply:
you mean above the yaml editor directly ?